server/game/effects.js

```diff
@@ -1141,6 +1141,26 @@ const Effects = {
                 game.skipPhase[name] = false;
             }
         };
+    },
+    cannotSelectSchemes: function() {
+        return {
+            apply: function(player) {
+                player.cannotSelectSchemes = true;
```

Comment:
This should work fine for now, but given all the things that are coming that interact with the plot deck I'm wondering if a more general solution would be justified. Don't have a list of those memorized so not sure if it's beneficial to anything coming. But you could do this more generally as:
```
this.persistentEffect({
  condition: () => this.game.currentPhase === 'plot',
  effect: ability.effects.cannotRevealPlot(plot => plot.hasTrait('Scheme')
});
```

I also worry somewhat about the 'Scheme cards are not considered to be in your plot deck' aspect during the plot phase in terms of future interactions with other cards, but we can cross that bridge when we come to it. I couldn't even begin to guess what the future implications are.

Reply:
A possible complication in this regard could arise when we convert The Crow is a Tricksy Bird to use a target prompt, as it would show even the Scheme plots of the Rains player, which it shouldn't. We could obviously work around that by checking if the opponent has Rains as their agenda on The Crow, but that seems a little iffy...

Reply:
Funny, before I read this, but after our earlier discussions, I was thinking about changing it to be exactly that :)

I'll see what I can do

Reply:
@DukeTax my changes don't fix your "Being able to see your opponents schemes" issue, but let's worry about that if and when we convert that to a target prompt :)